src/ui/Ui.java

```diff
@@ -0,0 +1,132 @@
+package ui;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import task.Deadline;
+import task.Task;
+
+
+
+/**
+ * Handles user interactions in the Giorgo application.
+ */
+public class Ui {
+
+    /**
```

Comment:
You actually went to the point of adding JavaDoc comments to EVERY SINGLE METHOD. Nice.

Reply:
Thanks!